x-pack/plugins/enterprise_search/public/applications/app_search/components/create_engine/create_engine.tsx

```diff
@@ -43,6 +43,7 @@ export const CreateEngine: React.FC = () => {
   const { name, rawName, language } = useValues(CreateEngineLogic);
   const { setLanguage, setRawName, submitEngine } = useActions(CreateEngineLogic);
 
+  // TODO these need to come from AppLogic and/or the Enterprise Search server
```

Comment:
If you're going commit by commit, I remove this in https://github.com/elastic/kibana/pull/89156/commits/04bdfbc156b3587dec130651bb5cab8790458046#diff-529e95b2a96ac2a4737364f8304d2c07b6095cb2378c51fb63a329faf8d5b86aL46-L51